resources/views/roles/index.blade.php

```diff
@@ -77,10 +77,19 @@ function onVisibilityChange() {
             // Page is visible, set a timeout to hide the flash message
             setTimeout(hideFlashMessage, 5000); // 5000 milliseconds = 5 seconds
 
-            // Event listener for visibility change
-            document.addEventListener('visibilitychange', onVisibilityChange);
         }
 
+        // Initialisierung - beim Laden prüfen, ob die Seite sichtbar ist
+        if (!document.hidden) {
+            // Wenn die Seite beim Laden sichtbar ist, Timeout sofort setzen
+            setTimeout(hideFlashMessage, 5000);
+        }
```

Comment:
Diese Funktionalität wird bereits in dem Aufruf von `onVisibilityChange()` ein paar Zeilen später umgesetzt. Bitte entferne diese Zeilen.